### PR TITLE
feat, fix, style : see commits.

### PR DESCRIPTION
**Pull request by AlucardTN**

### CODE REVIEW

**Comment by AlucardTN** [comment]
DesignActivity is high file even i used CRLF and design want to detect new lines or changes, btw i only changed things of compile from line proguard

**Comment by AlucardTN** [comment]
also, hosni forget to rename hes methods, so i fixed them

**Comment by AlucardTN** [comment]
thats all, still some bugs about incremental build

**Comment by JavkhlanK** [comment]
Are the files you committed with LF separators or CRLF ones? I can't quite tell on `R8Executor`

**Comment by AlucardTN** [comment]
i used LF (\n)

**Review by JavkhlanK — CHANGES_REQUESTED** [review]

File: app/src/main/java/mod/hey/studios/build/BuildSettings.java [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 import android.os.Environment;
 
+import com.android.tools.r8.w.S;
```
[the remark this answers is not in the document]
Comment (AlucardTN) [reply app/src/main/java/mod/hey/studios/build/BuildSettings.java:5]: i the first one, i mean DEX not FEX, and i the second one, this import is with a fault, you can remove it

[the remark this answers is not in the document]
Comment (JavkhlanK) [reply app/src/main/java/mod/hey/studios/build/BuildSettings.java:5]: Could you fix these yourself and test changes out again? I don't want to edit your changes right now from the web without testing them...

[the remark this answers is not in the document]
Comment (AlucardTN) [reply app/src/main/java/mod/hey/studios/build/BuildSettings.java:5]: Am I lying to you when i say " i have tested, also Ilyasse & Tyron tested it.

[the remark this answers is not in the document]
Comment (AlucardTN) [reply app/src/main/java/mod/hey/studios/build/BuildSettings.java:5]: edit them from the web directly, I'll after 1 hours send git the repo again and send you what working and what's not working later

[the remark this answers is not in the document]
Comment (JavkhlanK) [reply app/src/main/java/mod/hey/studios/build/BuildSettings.java:5]: what if some file was referencing the constant in `BuildSettings` with the wrong name and I missed it, I meant you should rename it to `DEX_...` and see if it still compiles

[the remark this answers is not in the document]
Comment (AlucardTN) [reply app/src/main/java/mod/hey/studios/build/BuildSettings.java:5]: only `DesignActivity` and `BuildSettingsDialog` reference to that, just edit from web directly.

[the remark this answers is not in the document]
Comment (JavkhlanK) [reply app/src/main/java/mod/hey/studios/build/BuildSettings.java:5]: hmmm okay

[the remark this answers is not in the document]
Comment (JavkhlanK) [reply app/src/main/java/mod/hey/studios/build/BuildSettings.java:5]: please try to fix changes people request yourself next time

[the remark this answers is not in the document]
Comment (AlucardTN) [reply app/src/main/java/mod/hey/studios/build/BuildSettings.java:5]: wdym

[the remark this answers is not in the document]
Comment (JavkhlanK) [reply app/src/main/java/mod/hey/studios/build/BuildSettings.java:5]: like why do i have to edit a PR by you

**Pull request closed without merging**